### PR TITLE
Accessibility: Basic keyboard navigation + show focus ring on only keyboard interaction

### LINKED ISSUE #3: Make the DayNightSwitch component accessible
https://github.com/kitze/twizzy-landing/tree/master/src/components/DayNightSwitch

It's currently using only divs and it's completely inaccessible.

### PR DESCRIPTION
**Pull request by ganapativs**
Fixes #3

### CODE REVIEW

**Comment by ganapativs** [comment]
Let me know if something is missing or needs to be updated.

File: src/components/ToggleCount/index.js [hunk 1]
```diff
@@ -47,7 +45,19 @@ function ToggleCount({ count, onTweet }) {
...
+            role="button"
+            tabIndex={0}
+            onKeyPress={e => {
+              if (e.which === 13) {
+                onTweet(e);
+              }
+            }}
+          >
+            {' '}
+            Tweet progress{' '}
```
Comment (ganapativs) [inline src/components/ToggleCount/index.js]: Prettier auto formatted.

**Comment by kitze** [comment]
Hey @ganapativs thanks for this PR! I just tried it and it works ok except the focus ring gets lost after the user presses enter once. Can we keep the focus ring? Also it would be nice if it works with the space key. 

Thanks!

**Comment by ganapativs** [comment]
@kitze Yes, I'll try to figure out why focus ring is getting lost(may be because of re-rendering/re-mount? 🤔), Space bar implementation is new to me. Will see if I can fix it.

**Comment by kitze** [comment]
Thank you @ganapativs !

File: src/components/DayNightSwitch/index.js [hunk 2]
```diff
@@ -10,19 +10,32 @@ import * as S from './styles';
...
-          <S.Star index={3} size={1} x={12} y={18} />
-          <S.Star index={4} size={1} x={15} y={10} />
-          <S.Star index={5} size={1} x={19} y={4} />
-          <S.Star index={6} size={2} x={22} y={14} />
-        </S.Stars>
-        <S.Circle>
-          <S.Moon icon={faMoon} />
-        </S.Circle>
-      </S.DayNightSwitch>
+      <div>
```
Comment (ganapativs) [inline src/components/DayNightSwitch/index.js]: Had to wrap it in div and span to keep focus ring as theme change was removing `focus-visible` from `DayNightSwitch` element.

**Comment by ganapativs** [comment]
PR has been update to support spacebar and to preserve focus ring on DayNightSwitch.

**Comment by ganapativs** [comment]
Fun part 😛

kapture 2018-11-23 at 19 49 29

**Comment by kitze** [comment]
Thanks @ganapativs, great job! 🙌️

**Pull request merged**